FRW/Modules/Wallet/FlowWalletKit/SecureEnclaveMigration.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  SecureEnclaveMigration.swift
+//  FRW
+//
+//  Created by cat on 2024/9/10.
+//
+
+import Foundation
+import KeychainAccess
+import FlowWalletKit
+import CryptoKit
+
+struct SecureEnclaveMigration {
+    private static var service: String = "com.flowfoundation.wallet.securekey"
+    private static var userKey: String = "user.keystore"
+    
+    static func start() {
+        let keychain = Keychain(service: service)
+        guard let data = try? keychain.getData(userKey) else {
+            print("[Migration] SecureEnclave get value from keychain empty,\(service)")
+            return
+        }
+        guard let users = try? JSONDecoder().decode([StoreInfo].self, from: data) else {
+            print("[Migration] SecureEnclave list decoder failed ")
+            return
+        }
+        let startAt = CFAbsoluteTimeGetCurrent()
+        var finishCount = 0
+        for item in users {
+            if let privateKey = try? SecureEnclave.P256.Signing.PrivateKey(dataRepresentation: item.publicKey) {
+                let secureKey = SecureEnclaveKey(key: privateKey, storage: SecureEnclaveKey.KeychainStorage)
```

Comment:
Can we add the canSign check in here? If the key is invalid, we shouldn't migrate them